### PR TITLE
Release 1.6.18

### PR DESCRIPTION
**Pull request by vmalaviya-splunk**
### Bug fixes
* [#405](https://github.com/splunk/splunk-sdk-python/pull/405) Fix searchcommands_app example
* [#406](https://github.com/splunk/splunk-sdk-python/pull/406) Fix mod inputs examples
* [#407](https://github.com/splunk/splunk-sdk-python/pull/407) Fixed issue with Streaming and Generating Custom Search Commands dropping fields that aren't present in the first row of results. More details on how to opt-in to this fix can be found here: 
https://github.com/splunk/splunk-sdk-python/blob/develop/README.md#customization [ [issue#401](https://github.com/splunk/splunk-sdk-python/issues/401) ]

### Minor changes
* [#408](https://github.com/splunk/splunk-sdk-python/pull/408) Add search mode example
* [#409](https://github.com/splunk/splunk-sdk-python/pull/409) Add Support for authorization tokens read from .splunkrc [ [issue#388](https://github.com/splunk/splunk-sdk-python/issues/388) ]
* [#413](https://github.com/splunk/splunk-sdk-python/pull/413) Default kvstore owner to nobody [ [issue#231](https://github.com/splunk/splunk-sdk-python/issues/231) ]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+### Bug fixes
+* [#405](https://github.com/splunk/splunk-sdk-python/pull/405) Fix searchcommands_app example
+* [#406](https://github.com/splunk/splunk-sdk-python/pull/406) Fix mod inputs examples
+* [#407](https://github.com/splunk/splunk-sdk-python/pull/407) Modified Streaming and Generating Custom Search Command
+
+### Minor changes
+* [#408](https://github.com/splunk/splunk-sdk-python/pull/408) Add search mode example
+* [#409](https://github.com/splunk/splunk-sdk-python/pull/409) Add Support for authorization tokens read from .splunkrc
+* [#413](https://github.com/splunk/splunk-sdk-python/pull/413) Default kvstore owner to nobody
```
Comment (fantavlik) [inline CHANGELOG.md:13]: Can we reference the issue as well so they are linked? https://github.com/splunk/splunk-sdk-python/issues/231

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Splunk Enterprise SDK for Python Changelog
 
+## Version 1.6.18
+
+### Bug fixes
+* [#405](https://github.com/splunk/splunk-sdk-python/pull/405) Fix searchcommands_app example
+* [#406](https://github.com/splunk/splunk-sdk-python/pull/406) Fix mod inputs examples
+* [#407](https://github.com/splunk/splunk-sdk-python/pull/407) Modified Streaming and Generating Custom Search Command
```
Comment (fantavlik) [inline CHANGELOG.md:8]: This probably needs edits to make it more clear what was fixed - `Modified Streaming and Generating Custom Search Command` probably wasn't a good title for that PR. Can we change this to:

> Fixed issue with Streaming and Generating Custom Search Commands dropping fields that aren't present in the first row of results. More details on how to opt-in to this fix can be found here: https://github.com/splunk/splunk-sdk-python/blob/develop/README.md#customization

**Review by fantavlik — APPROVED** [review]
Suggested some changes to the CHANGELOG but feel free to release once those are made

File: CHANGELOG.md (hunk 1 above)
Comment (vmalaviya-splunk) [reply CHANGELOG.md:13]: Sure!

File: CHANGELOG.md (hunk 2 above)
Comment (vmalaviya-splunk) [reply CHANGELOG.md:8]: Yes, this makes more sense. I'll make the change.

**Pull request merged**